test_execution_shutdown_failure: error: This error was triggered on purpose.
test_execution_shutdown_failure: error: This error was also triggered on purpose.
